package.json

```diff
@@ -2,24 +2,39 @@
   "name": "dredd-transactions",
   "version": "0.0.0-semantically-released",
   "description": "Compiles HTTP Transactions (Request-Response pairs) from API description document",
-  "main": "lib/index.js",
+  "main": "dist/index.js",
   "engines": {
     "node": ">= 4"
   },
   "scripts": {
-    "lint": "scripts/commitlint.sh && coffeelint src",
-    "build": "coffee -b -c -o lib/ src/",
-    "test": "mocha \"test/**/*-test.coffee\"",
-    "test:coverage": "scripts/coverage.sh",
-    "prepublish": "npm run build",
+    "build": "babel src --out-dir ./dist/src",
+    "clean": "rimraf dist",
+    "copy:test": "node scripts/copy.js",
     "coveralls": "scripts/coveralls.sh",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "lint": "scripts/commitlint.sh && eslint src",
+    "prepublish": "npm run build",
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "test": "npm run clean && npm run copy:test && npm run build && mocha \"./dist/test/**/*-test.coffee\" --opts \"./mocha.opts\"",
```

Comment:
Why mocha.opts are moved out of the tests directory? Is it related to the facts we need to play around with tests since they're still in coffee?

Reply:
`mocha.opts` gets picked up by `mocha` automatically when located inside `test` folder - I needed to run it with different args for coverage. I'll sort it out.

Reply:
I'm okay to have it sorted out as a next step with removing CoffeeScript completely, if that helps.